typing/printtyp.ml

```diff
@@ -1656,11 +1656,13 @@ type syntactic_sig_item =
   { src: Types.signature_item; ghosts: Types.signature_item list }
 type rec_item_group =
   | Not_rec of syntactic_sig_item
-  | Rec_group of (Ident.t list * syntactic_sig_item list)
+  | Rec_group of ( (bool * Ident.t) list * syntactic_sig_item list)
```

Comment:
On the value-construction side, booleans are not very informative. It would make sense to use a proper record here.

Reply:
Fixed with an explicit record for identifiers that should be hidden.